wp-berry.php

```diff
@@ -15,6 +15,7 @@
 use WPBerry\Settings;
 
 require_once plugin_dir_path( __FILE__ ) . 'inc/settings.php';
+require_once plugin_dir_path( __FILE__ ) . 'inc/tickets.php';
 
 register_activation_hook( __FILE__, array( new Settings(), 'register_role' ) );
```

Comment:
Consider creating settings object once and using it with all registration hooks at once. Can call a single function for `registration_activation` and call all the necessary activation methods inside them. This might be out of scope of the ticket but would be good to refactor a small issue.